dist/node/index.js

```diff
@@ -0,0 +1,236 @@
+
```

Comment:
Can you not check the `dist` directory in at all? See [here](https://docs.google.com/document/d/1KALf-WokfvoNXxDOBruTrT6yki5hqfkj1f6OYYTlnEg/edit#bookmark=id.2ou7fsd4dngg) for how to configure `.gitignore` and `package.json`.